samples/outlook/99-preview-apis/close-async.yaml

```diff
@@ -1,7 +1,7 @@
 order: 3
 id: outlook-close-async
 name: Close the current message and discard changes (Message Compose)
-description: Close the current message and discard the changes when specified.
+description: Close the current message and discard any unsaved changes when specified.
```

Comment:
```suggestion
description: Closes the current message and discards any unsaved changes when specified.
```

Reply:
Thanks for the suggestion. I'll retain the current wording to keep it consistent with other snippets.